spot-ml/src/main/scala/org/apache/spot/SuspiciousConnects.scala

```diff
@@ -36,7 +36,7 @@ object SuspiciousConnects {
     parser.parse(args, SuspiciousConnectsConfig()) match {
       case Some(config) =>
 
-        Logger.getLogger("org").setLevel(Level.OFF)
+        Logger.getLogger("org").setLevel(Level.WARN)
```

Comment:
Based on Vartikas recommendation, we should leave `org` at least at WARN level. If there are no warnings, the output looks exactly the same.